modules/fluid_properties/test/tests/sodium/sodium.i

```diff
@@ -0,0 +1,111 @@
+# @Requirement F4.70
+#
+# Parameters for this case are pulled from various places to emulate a typical SFR. Flow rate is
+# assumed to be 8 m/s and density of sodium is about 880 kg/m3, so mass flux is 7040 kg/m2-s.
+# Temperature rise is about 140 K.
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  xmin = 3.1e-3
+  xmax = 3.7e-3
+  ymin = 0
+  ymax = 1
+  nx = 4
+  ny = 20
+[]
+
+[Problem]
+  coord_type = RZ
+[]
+
+[Modules]
+  [./FluidProperties]
+    [./sodium]
+      type = SodiumProperty
+    [../]
+  [../]
+[]
+
+[Variables]
+   [./temperature]
+    initial_condition = 700
+  [../]
+[]
+
+[Kernels]
+  [./heat]
+    type = HeatConduction
+    variable = temperature
+  [../]
+[]
+
+[BCs]
+  [./clad_inner_surface]
+    type = NeumannBC
+    boundary = left
+    value = 2387097 # 2e6 W/m2 heat flux at outside of clad * area ratio of outside/inside of clad.
+    variable = temperature
+  [../]
+[]
+
+[CoolantChannel]
+  [./clad_outer_surface]
+    boundary          = right
+    variable          = temperature
+    inlet_temperature = 643       # 370 C
+    inlet_massflux    = 7040      # kg/m^2-sec
+    rod_diameter      = 7.4e-3    # m
+    rod_pitch         = 8.9e-3    # m
+    coolant_material = sodium
+    number_axial_zone = 20
+    direction = y
+    fluid_properties = sodium
+  [../]
+[]
+
+[Materials]
+  [./clad_thermal]
+    type = HeatConductionMaterial
+    block = 0
+    thermal_conductivity = 16.0  # W/m-K
+    specific_heat = 330.0        # J/kg-K
+  [../]
+  [./density]
+    type = Density
+    block = 0
+    density = 6551.0             # kg/m^3
+  [../]
+[]
+
+# Copy the coolant_temperature property to an aux var.
+[AuxVariables]
+  [./coolant_temperature]
+    family = monomial
+    order = constant
+  [../]
+[]
+[AuxKernels]
+  [./copy_coolant]
+    type = MaterialRealAux
+    variable = coolant_temperature
+    property = coolant_temperature
+    boundary = right
+    execute_on = 'initial timestep_end'
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+  solve_type = 'PJFNK'
+
+  nl_abs_tol = 1e-8
+
+  end_time = 2
+  dt = 0.5
+[]
+
+# Define output file(s)
+[Outputs]
+  exodus = true
+[]
```

Comment:
This test cannot possibly run in fluid properties module. You are using app-specific syntax and classes from modules that are not dependencies of fluid properties. Go compile fluid properties module on your machine and execute `run_tests`. You should see the test fail.

Also, if you have unit tests for this new class (which would be preferred and easier actually), put them in `moose/unit`. That's where the classes from fluid properties module are unit-tested. You will see several there, so just follow the pattern.

Then you do not need this at all.